test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -211,6 +211,10 @@ export default class BaseFormObject {
   }
 
   getCancelButtonLabel() {
+    // in v3 buttons dont have a value prop
+    if (userVariables.v3) {
+      return this.el.find(CANCEL_BUTTON_SELECTOR).textContent;
```

Comment:
Does this also work on the gen2 test?

Reply:
It doesn't because in Gen 2 the button is actually an `input` so `textContent` does not work, whereas in gen 3 it is a button (where `.value` does not work like in Gen 2).